### PR TITLE
Release v1.0.0-alpha10

### PR DESCRIPTION
**Pull request by deb1990**
### Fixes
1. Activity Feed does not load when a Payment Type activity is present.
2. Month Navigation does not show up on Activity Feed.

### CODE REVIEW

File: ang/civicase/activity/feed/directives/activity-feed.directive.js [hunk 1]
```diff
@@ -479,6 +472,26 @@
       });
     }
 
+    /**
+     * Prepares Activity Params
```
Comment (swastikpareek) [inline ang/civicase/activity/feed/directives/activity-feed.directive.js]: Why do we want to explicitly say that `Prepares Activity Params` when the name `prepareActivityParams` itself says that. 
Can't we just change it to -
```
    /**
     * Adds Action Links chained Api call
     *
     * @param {Object} returnParams
     * @param {Object} params
     *
     * @return {Array}
     */
```

File: api/v3/Activity/Getactionlinks.php [hunk 2]
```diff
@@ -1,51 +1,65 @@
 <?php
 
+/**
+ * @file
+ * Activity.getactionlinks file.
```
Comment (swastikpareek) [inline api/v3/Activity/Getactionlinks.php]: Again. What extra info this `Activity.getactionlinks file.` text adds to the documentation that the file name doesn't have.
What's the need of having it at the first place.

File: api/v3/Activity/Getactionlinks.php [hunk 3]
```diff
@@ -1,51 +1,65 @@
...
 /**
- * Activity.getactionlinks API specification (optional)
- * This is used for documentation and validation.
+ * Activity.getactionlinks API specification (optional).
  *
- * @param array $spec description of fields supported by this API call
- * @return void
- * @see http://wiki.civicrm.org/confluence/display/CRMDOC/API+Architecture+Standards
+ * @param array $spec
+ *   description of fields supported by this API call.
```
Comment (swastikpareek) [inline api/v3/Activity/Getactionlinks.php]: Shouldn't `description` be `Description` rather?

File: api/v3/Activity/Getactionlinks.php [hunk 4]
```diff
@@ -1,51 +1,65 @@
...
  * added by various extensions via hooks. Action links differ per activity
- * as some logic may determine which links are available for a particular activity.
+ * as some logic may determine which links are available for a particular
+ * activity.
  *
  * @param array $params
+ *   Params.
  *
  * @return array
+ *   Links
```
Comment (swastikpareek) [inline api/v3/Activity/Getactionlinks.php]: add `.` if we have added for the `Params.`

**Comment by deb1990** [comment]
@swastikpareek As this is a release pr, not fixing any comments for already approved code, but we will keep this in mind for future. thanks.

**Pull request merged**